core/src/main/scala/com.mediative.amadou/SparkJob.scala

```diff
@@ -65,4 +65,23 @@ trait SparkJob extends Logging {
    * measured. For example, many Spark operations are lazy.
    */
   def stage[T](stage: Stage)(f: => T): T = f
+
+  implicit class SparkReadOps(self: DataFrameReader) {
```

Comment:
If they live in `SparkJob` they won't automatically take effect when extending SparkJob. Also anyvals are not allowed inside traits.